api/edge_api/identities/events.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+import boto3
+from django.conf import settings
+
+events_client = None
+if settings.IDENTITY_MIGRATION_EVENT_BUS:
+    events_client = boto3.client("events")
+
+
+def send_migration_event(project_id: int):
+    event = {
+        "EventBusName": settings.IDENTITY_MIGRATION_EVENT_BUS,
+        "Source": "flagsmith.api.migrate",
+        "DetailType": "Migrate Identity to dynamodb",
```

Comment:
I'm not sure exactly where this gets used but shouldn't it be something along the lines of `f"Migrate all identities for project {project_id} to dynamodb"`?

Reply:
Ah, no, this needs to be static
```
detail-type
    Identifies, in combination with the source field, the fields and values that appear in the detail field.
```
ref: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/CloudWatchEventsandEventPatterns.html

Reply:
Ok, that's fine then. But we should at least correct it to use 'identities' plural, right? It's not migrating a single identity unless I'm mistaken?

Reply:
Yep, that's correct. Will update.